vars/dockerExecuteOnKubernetes.groovy

```diff
@@ -430,6 +447,22 @@ private List getContainerList(config) {
     return result
 }
 
+private Map getResources(String imageName, Map config) {
```

Comment:
```suggestion
private Map getResources(String ContainerName, Map config) {
```
container name is different from image name. Hence, I would prefer to get the resources for the given container name.

Reply:
You are right. Understood. Changed similar to your suggestion. We are not talking about properties for an `image`, but for a `container`. I will change that, but according to widely used java naming conventions with lower case "C".